src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -258,7 +259,8 @@ foam.CLASS({
       label: 'Refresh',
       toolTip: 'Refresh Table',
       icon: 'images/refresh-icon-black.svg',
-      isAvailable: function(config) {
+      isAvailable: function(config, displayWidth) {
+        if ( displayWidth.minWidth < foam.u2.layout.DisplayWidth.MD.minWidth ) return false;
```

Comment:
This would remove the refresh button completely. L:397 of this file we set a limit on the number of buttons that appear on the browser view and the rest collapse into an overflow menu. Better to make that number 1 instead of 3 based on displayWidth